jenkins/open-mpi-build-script.sh

```diff
@@ -179,6 +179,9 @@ fi
 # ./configure --help.  In prep for 5.0, we added a developer
 # requirement for pandoc to build (not required for dist tarballs),
 # with an explicit option to disable.
+# JMS: The requirement will shortly change from pandoc to Sphinx.
+# Keep pandoc for now; when Sphinx is brought to the v5.0.x branch, we
```

Comment:
We run CI tests on branches prior to 5.0.x, so we can't remove this block for some time.  That said, it needs to change.  I think the logic becomes:

1) if no --disable-man-pages option to configure, do nothing. (this is a pre-pandoc build)
2) if docs/requirements.txt exists, do nothing (this is a 5.0.x or later build, so configure will autoselct forward whether or not Sphinx is found).
3) if pandoc is found, do nothing (this is a pandoc-era build, and we have pandoc)
4) else, add --disable-man-pages to configure (this is a pandoc-era build, and we don't have pandoc, so we need to add a configure argument to let the build succeed).